README.md

```diff
@@ -1,4 +1,4 @@
-#python-crpapi
+# python-crpapi
```

Comment:
If there isn't a space between the text and pound signs they just merge into a single unformatted word.